examples/with-tigris/db/models/todoItems.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Linter was reporting following errors:
```
   2:3   warning  'Field' is defined but never used             @typescript-eslint/no-unused-vars
   3:3   warning  'PrimaryKey' is defined but never used        @typescript-eslint/no-unused-vars
   4:3   warning  'TigrisCollection' is defined but never used  @typescript-eslint/no-unused-vars
   6:3   warning  'TigrisDataTypes' is defined but never used   @typescript-eslint/no-unused-vars
  10:14  warning  'TodoItem' is defined but never used          @typescript-eslint/no-unused-vars
```
However, these are decorators actually being used in the same file. It could be that the project is using outdated ESLint that does not recognize decorators as of yet.

Reply:
This can be set in the [root .eslintignore](https://github.com/vercel/next.js/blob/canary/.eslintignore) instead so this comment isn't needed.